packages/origin-247-certificate/src/onchain-certificate/onChainCertificateFacade.ts

```diff
@@ -2,12 +2,17 @@ import { IBlockchainProperties } from '@energyweb/issuer';
 import { Injectable } from '@nestjs/common';
 import { BlockchainProperties, BlockchainPropertiesService } from './blockchain-properties.service';
 
+type DeployParameters = {
```

Comment:
this could be imported from @energyweb/issuer

Reply:
implemented on [f4a209b](https://github.com/energywebfoundation/origin-247-sdk/pull/104/commits/f4a209bc63a412fb229c376e64d8a1496a6832c5)